### ISSUE TITLE
False positive on match with object's attribute

### ISSUE THREAD

**Issue by Akomer** [charliermarsh/ruff#1414]
I got a false positive SyntaxError in case i am using the match on an object's attribute:

```python
class A:
    var1 = 5


def false_positive_case():
    a = A()
    match a.var1:
        case 5:
            print('easy match')
        case _:
            print('sad')


def okay_case():
    a = [1, 2, 3]
    match a:
        case [v1, v2, *_]:
            print('easy matcH:', v1, v2)


if __name__ == '__main__':
    false_positive_case()
    okay_case()
```

```shell
ruff ruff_match.py
ruff_match.py:7:12: E999 SyntaxError: invalid syntax. Got unexpected token 'a'
Found 1 error(s).
```

versions:
Python 3.10.6
ruff 0.0.196

**Comment by charliermarsh** [comment]
Sadly, we don't yet support structural pattern matching (see: #282). It's the one language feature that we're still working on.

**Comment by charliermarsh** [comment]
Fixed as of v0.0.250 which included `match` statement support.
